src/js/components/telephone-input.jsx

```diff
@@ -0,0 +1,13 @@
+import ReactTelInput from 'react-telephone-input';
+
+export class TelephoneInput extends ReactTelInput {
+    getFlagStyle = () => {
```

Comment:
maybe it has to do with the fact that the original class is using React.createClass and not a ES6 class... that's fine then, use fat arrow in this case